security/calico-enterprise/federation.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Federation
-description: Federated services provide service discovery of remote pods in another cluster. 
+description: Simplify the creation and maintenance of security controls using federated tiers and policies.
```

Comment:
This one still doesn't feel right to me.  This page covers both federated identity of endpoints (pods) and Kubernetes services; plus federated policies and tiers.  Neither description captures this very well.

Reply:
Not sure I still have it right...see what you think.

Reply:
"security for tiered policies" isn't quite right.  It would be more like "security with tiered policies".  (Federation doesn't secure tiered policies; it allows you to use tiered policies to do security.)

Reply:
Fixed.